osgar/drivers/eduro.py

```diff
@@ -176,31 +179,29 @@ def check_restarted_modules(self, module_id, status):
                 self.dist_right_diff = 0
 
     def process_packet(self, packet, verbose=False):
-        if len(packet) >= 2:
-            msg_id = ((packet[0]) << 3) | (((packet[1]) >> 5) & 0x1f)
```

Comment:
I am so happy to see this line go away from eduro.py! :dancer:

Reply:
:)